docs/reference/elasticsearch/rest-apis/reindex-indices.md

```diff
@@ -0,0 +1,630 @@
+---
+navigation_title: "Reindex indices"
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html
+applies_to:
+  stack: all
+---
+
+# Reindex indices examples
+
+::::{admonition} New API reference
+For the most up-to-date API details, refer to [Document APIs](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-reindex).
+::::
+
+## Running reindex asynchronously [docs-reindex-task-api]
+
+If the request contains `wait_for_completion=false`, {{es}} performs some preflight checks, launches the request, and returns a `task` you can use to cancel or get the status of the task. {{es}} creates a record of this task as a document at `_tasks/<task_id>`.
+
+## Reindex from multiple sources [docs-reindex-from-multiple-sources]
+
+If you have many sources to reindex it is generally better to reindex them one at a time rather than using a glob pattern to pick up multiple sources.
+That way you can resume the process if there are any errors by removing the partially completed source and starting over.
+It also makes parallelizing the process fairly simple: split the list of sources to reindex and run each list in parallel.
+
+One-off bash scripts seem to work nicely for this:
+
+```bash
+for index in i1 i2 i3 i4 i5; do
+  curl -HContent-Type:application/json -XPOST localhost:9200/_reindex?pretty -d'{
+    "source": {
+      "index": "'$index'"
+    },
+    "dest": {
+      "index": "'$index'-reindexed"
+    }
+  }'
+done
+```
+
+## Throttling [docs-reindex-throttle]
+
+Set `requests_per_second` to any positive decimal number (for example, `1.4`, `6`, or `1000`) to throttle the rate at which the reindex API issues batches of index operations.
+Requests are throttled by padding each batch with a wait time.
+To disable throttling, set `requests_per_second` to `-1`.
+
+The throttling is done by waiting between batches so that the `scroll` that the reindex API uses internally can be given a timeout that takes into account the padding. The padding time is the difference between the batch size divided by the `requests_per_second` and the time spent writing. By default the batch size is `1000`, so if `requests_per_second` is set to `500`:
+
+```txt
+target_time = 1000 / 500 per second = 2 seconds
+wait_time = target_time - write_time = 2 seconds - .5 seconds = 1.5 seconds
+```
+
+Since the batch is issued as a single `_bulk` request, large batch sizes cause {{es}} to create many requests and then wait for a while before starting the next set. This is "bursty" instead of "smooth".
+
+## Rethrottling [docs-reindex-rethrottle]
+
+The value of `requests_per_second` can be changed on a running reindex using the `_rethrottle` API. For example:
+
+```console
```

Comment:
consider restoring the commented test snippets from the original `asciidoc` replacing `//` with `%`